server.go

```diff
@@ -17,16 +17,19 @@ import (
 	"gopkg.in/juju/charmstore.v5-unstable/internal/charmstore"
 	"gopkg.in/juju/charmstore.v5-unstable/internal/legacy"
 	"gopkg.in/juju/charmstore.v5-unstable/internal/v4"
+	"gopkg.in/juju/charmstore.v5-unstable/internal/v5"
 )
 
 // Versions of the API that can be served.
 const (
 	V4     = "v4"
+	V5     = "v5"
```

Comment:
Purely pedantic remark, can we keep descending order here? Or ascending. Just not mixed :)